src/components/Navigation/Navigation.tsx

```diff
@@ -1,31 +1,66 @@
-import React, { FC, useCallback } from 'react';
+import React, { FC, useCallback, useState, MouseEvent } from 'react';
 import { useHistory } from 'react-router-dom';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faChevronLeft } from '@fortawesome/free-solid-svg-icons';
+import { faChevronLeft, faEllipsisH } from '@fortawesome/free-solid-svg-icons';
 import { IconButton } from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
```

Comment:
This can just be 
`import { IconButton, MenuItem, Menu } from '@material-ui/core';`

Reply:
Addressed